nilearn/image/tests/test_resampling.py

```diff
@@ -269,7 +269,12 @@ def test_resampling_error_checks():
         rot_img = resample_img(img_binary,
                                target_affine=rot,
                                interpolation='linear')
-
+    
+    img_no_sform = Nifti1Image(data, affine)
+    img_no_sform.set_sform(None)
+    with pytest.warns(Warning, match="The provided image has no sform"):
+        no_sform = resample_img(img_no_sform,
```

Comment:
You don't need to assign the return value to a variable here:

```suggestion
        resample_img(img_no_sform,
```